### DISCUSSION TITLE
DateTimeInput / free input DateTimePicker

### DISCUSSION THREAD

**Discussion by finnharbeke** [mantinedev/mantine#4163] [Q&A]
Hey is there any possibility to let a DateTimePicker allow free input as does DateInput?

I didn't find a component prop to enable it and there's no DateTimeInput component either.

Why I'd use it: on the computer I find free inputs much easier / faster and want to type the datetime, on the phone it's a lot user-friendlier if the input is not free, ie like the existing DateTimePicker.

**Comment by rtivital** [comment]
You can use `<TextInput type="datetime-local" />`

**Reply by finnharbeke** [reply]
thanks for the reply! i don't really like the native picker, so i guess i'll go with the combination of a DateInput and a TimeInput component.
